oscarpolanco/movie-project
# Separate logic of the moviePage component

[Issue: Improve the project folder logic structure](https://github.com/oscarpolanco/movie-project/issues/9)

Separate the logic of the `moviePage` component into separate components and function helper

## Step to test:
1. `npm install`
2. add a `.env`  file on the root of the project
3. Add the following constant (you can get your key [here](https://www.themoviedb.org/settings/api)):
`MOVIE_API_KEY=**********`
4. `npm run dev`
5. Go to [localhost:1234](http://localhost:1234/)
6. You should see 20 movies
7. Scroll to the bottom and you should see that you have 20 more movies
